lib/phlex/rails/form_builder.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+class Phlex::Rails::FormBuilder < Phlex::Rails::Decorator
+	define_method :__caller_locations__, ::Object.instance_method(:caller_locations)
+
+	def id(...) = @object.id(...)
+	def to_model(...) = @object.to_model(...)
+	def field_id(...) = @object.field_id(...)
+	def field_name(...) = @object.field_name(...)
+	def to_partial_path(...) = @object.to_partial_path(...)
```

Comment:
These are just here for performance. All missing methods will be delegated anyway.